HackYourFuture-CPH/simply-name.it
# Welcome page

# Description

The Welcome Page container, as per the wireframes, uses firebase to authenticate users with Google provider using authentication hook and authentication route, delivering the user information and firebase token. 
Fixes #77 #90 

# How to test?
Design of Welcome Page:
run npm run storybook and see the Welcome Page container on the Large Mobile screen setting. 

Firebase authentication:
npm run dev on localhost:3000/welcome
# Checklist
